ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoColorProviding.m

```diff
@@ -14,39 +14,39 @@ @interface ObjectiveCDemoColorProviding () <MSFColorProviding>
-                                                          darkHighContrast:nil
-                                                              darkElevated:nil
-                                                  darkElevatedHighContrast:nil];
+    UIColor *dynamicColor = [[UIColor alloc] initWithLight:lightColor
+                                         lightHighContrast:nil
+                                             lightElevated:nil
+                                 lightElevatedHighContrast:nil
+                                                      dark:darkColor
+                                          darkHighContrast:nil
+                                              darkElevated:nil
+                                  darkElevatedHighContrast:nil];
 
-    return [[UIColor alloc] initWithDynamicColor:dynamicColor];
+    return dynamicColor;
 }
 
 - (UIColor *)brandForegroundColor {
-    MSFColorValue *lightColor = [MSFGlobalTokens sharedColorForColorSet:MSFSharedColorSetsOrchid
-                                                                  token:MSFSharedColorsTokensShade30];
-    MSFColorValue *darkColor = [MSFGlobalTokens sharedColorForColorSet:MSFSharedColorSetsOrchid
-                                                                token:MSFSharedColorsTokensTint40];
-
-    MSFDynamicColor *dynamicColor = [[MSFDynamicColor alloc] initWithLight:lightColor
-                                                             lightHighContrast:nil
-                                                             lightElevated:nil
-                                                 lightElevatedHighContrast:nil
-                                                                      dark:darkColor
-                                                          darkHighContrast:nil
-                                                              darkElevated:nil
-                                                  darkElevatedHighContrast:nil];
-
-    return [[UIColor alloc] initWithDynamicColor:dynamicColor];
+    UIColor *lightColor = [MSFGlobalTokens colorForSharedColorSet:MSFGlobalTokensSharedColorSetOrchid
+                                                            token:MSFGlobalTokensSharedColorShade30];
+    UIColor *darkColor = [MSFGlobalTokens colorForSharedColorSet:MSFGlobalTokensSharedColorSetOrchid
+                                                           token:MSFGlobalTokensSharedColorTint40];
+
+    UIColor *dynamicColor = [[UIColor alloc] initWithLight:lightColor
+                                         lightHighContrast:nil
+                                             lightElevated:nil
+                                 lightElevatedHighContrast:nil
+                                                      dark:darkColor
+                                          darkHighContrast:nil
+                                              darkElevated:nil
+                                  darkElevatedHighContrast:nil];
```

Comment:
Is it possible to add another convenience init that just takes light and dark, so we don't need 6 `nil`s?